server/job/schedule.ts

```diff
@@ -32,63 +32,65 @@ export const startJobs = (): void => {
     cancelFn: () => jobPlexRecentSync.cancel(),
   });
 
-  // Run full plex sync every 24 hours
+  // Run full plex scan every 24 hours
   scheduledJobs.push({
-    id: 'plex-full-sync',
-    name: 'Plex Full Library Sync',
+    id: 'plex-full-scan',
+    name: 'Plex Full Library Scan',
     type: 'process',
     job: schedule.scheduleJob('0 0 3 * * *', () => {
-      logger.info('Starting scheduled job: Plex Full Sync', { label: 'Jobs' });
+      logger.info('Starting scheduled job: Plex Full Library Scan', {
+        label: 'Jobs',
+      });
       jobPlexFullSync.run();
     }),
     running: () => jobPlexFullSync.status().running,
     cancelFn: () => jobPlexFullSync.cancel(),
   });
 
-  // Run full radarr sync every 24 hours
+  // Run full radarr scan every 24 hours
   scheduledJobs.push({
-    id: 'radarr-sync',
-    name: 'Radarr Sync',
+    id: 'radarr-scan',
+    name: 'Radarr Scan',
     type: 'process',
     job: schedule.scheduleJob('0 0 4 * * *', () => {
-      logger.info('Starting scheduled job: Radarr Sync', { label: 'Jobs' });
+      logger.info('Starting scheduled job: Radarr Scan', { label: 'Jobs' });
       jobRadarrSync.run();
     }),
     running: () => jobRadarrSync.status().running,
     cancelFn: () => jobRadarrSync.cancel(),
   });
 
-  // Run full sonarr sync every 24 hours
+  // Run full sonarr scan every 24 hours
   scheduledJobs.push({
-    id: 'sonarr-sync',
-    name: 'Sonarr Sync',
+    id: 'sonarr-scan',
+    name: 'Sonarr Scan',
     type: 'process',
     job: schedule.scheduleJob('0 30 4 * * *', () => {
-      logger.info('Starting scheduled job: Sonarr Sync', { label: 'Jobs' });
+      logger.info('Starting scheduled job: Sonarr Scan', { label: 'Jobs' });
       jobSonarrSync.run();
     }),
     running: () => jobSonarrSync.status().running,
     cancelFn: () => jobSonarrSync.cancel(),
   });
 
-  // Run download sync
+  // Run download scan
   scheduledJobs.push({
-    id: 'download-sync',
-    name: 'Download Sync',
+    id: 'download-scan',
+    name: 'Download Scan',
```

Comment:
This is actually in sync!

Reply:
Doesn't "sync" infer it is two-way?  Don't we just scan and display the download status? 🤔

Reply:
Not necessarily? It just means the state is synchronized. Which it is.

Reply:
I suppose that's true.  But I still feel like it'd be clearer to have these current tasks which just fetch the current status of things be called "scans," and future jobs which do two-way synchronization to be called "syncs"?

(Just thinking ahead here.  IMO we should try to be deliberate and clear with language to minimize confusion around what these jobs actually do.)